src/Graph.cc

```diff
@@ -847,7 +847,7 @@ sea_dsa::Cell &sea_dsa::Graph::mkCell (const llvm::Value &u, const Cell &c)
 
     if (res->getRawOffset () != 0 && res->getNode ())
     {
-      if (res->getNode()->hasOnceUniqueScalar()) {
```

Comment:
I think this introduces a bug while fixing the other issue.
`hasOnceUniqueScalar` says that this node already had a scalar, and it was replaced by null for some reason. However, the unique scalar property is lost. `getUniqueScalar`, on the other hand, will be non-null only if there is an active scalar present. A better solution is to protect line 853!